### ISSUE TITLE
Why do you add both generated and content image ?

### ISSUE THREAD

**Issue by shamanez** [sebasvega95/neural-style-transfer#1]
In this [line](https://github.com/sebasvega95/neural-style-transfer/blob/master/loss.py#L53) you have added both content image and the generated image. But what you need to do is to get layer representations of both images. Maybe there is some kind of an axis error. 

Am I right?

**Comment by sebasvega95** [comment]
Hi! What that line does is taking the content batch images and their respective generated images and [concatenating](https://www.tensorflow.org/api_docs/python/tf/concat) them into a single batch to be fed to the VGG network. This is to get the layer representations of content and generated images in a single pass through the VGG network.

Once we get the layer representations of this batch, we [split](https://www.tensorflow.org/api_docs/python/tf/split) it accordingly to get the layer representations of content and generated images ([here](https://github.com/sebasvega95/neural-style-transfer/blob/master/loss.py#L63) and [here](https://github.com/sebasvega95/neural-style-transfer/blob/master/loss.py#L72)).

Is it clear now?

**Comment by shamanez** [comment]
Oh yes. Perfect. I am trying to train this from scratch. How long did it take? BTW your implementation is superb!

**Comment by sebasvega95** [comment]
Thanks! The time to train depends on your setup. I trained on a computer with two GPUs, a Titan X and a K40, and it takes about 2h40m for two epochs over the MSCOCO 2014 train set.
